main.sh

```diff
@@ -24,8 +24,10 @@
 #
 #  LOG_LEVEL=7 ./main.sh -f /tmp/x -d
 #
-# Licensed under MIT
-# Copyright (c) 2013 Kevin van Zonneveld (http://kvz.io)
+# Licensed under MIT with a less restrictive clause that excepts you of
```

Comment:
Commenting on the grammar only, I suggest replacing "excepts you of" with "exempts you from".